pkg/manager/impl/execution_manager.go

```diff
@@ -1019,10 +1021,32 @@ func (m *ExecutionManager) GetExecutionData(
 	if err != nil {
 		return nil, err
 	}
-	return &admin.WorkflowExecutionGetDataResponse{
+	response := &admin.WorkflowExecutionGetDataResponse{
```

Comment:
@katrogan one reason to do this is to avoid having to sign the URL, so can we just do a `HEAD`, check the size and if that is ok then do a protobuf get using flytestdlib.storage?

Reply:
or maybe i am misunderstanding, do we actually store the size of the payload in our database?

Reply:
until the UI transitions to using the new field, we should return both for backwards compatibility 
but good point on using flytestdlib, will update

Reply:
no, we don't store the size

Reply:
oh wait, and the storage client is using flytestdlib already